modules/update-service-create-service-cli.adoc

```diff
@@ -0,0 +1,75 @@
+[id="update-service-create-service-cli"]
+= Creating an Update Service using the CLI
+include::modules/common-attributes.adoc[]
+
+You can use the {product-title} CLI to create an Update Service using the {product-title} Update Service operator.
+
+== Procedure
+
+To create an Update Service using the {product-title} CLI:
+
+.. Configure the Update Service target namespace:
++
+[source,terminal]
+----
+$ NAMESPACE=FIXME-default-operand-namespace
+----
++
+This must match `targetNamespaces` from the OperatorGroup.
+
+.. Configure the name of the Update Service:
++
+[source,terminal]
+----
+$ NAME=example
+----
+
+.. Configure the local registry and repository for the release images:
++
+[source,terminal]
+----
+$ RELEASE_IMAGES=registry.example.com/openshift/release
+----
+
+.. Configure the local pullspec for the graph-data image:
++
+[source,terminal]
+----
+$ GRAPH_DATA_IMAGE=registry.example.com/openshift/graph-data:latest
+----
+
+.. Create an Update Service YAML file (for example, `update-service.yaml`):
++
+[source,terminal]
+----
+$ oc -n "${NAMESPACE}" create -f - <<EOF
+apiVersion: updateservice.openshift.io/v1
+kind: UpdateService
+metadata:
+  name: ${NAME}
+spec:
+  replicas: 2
+  releaseImages: ${RELEASE_IMAGES}  # FIXME: aspirational
+  graphDataImage: ${GRAPH_DATA_IMAGE}
+EOF
+----
+
+. Verify the Update Service:
+
+.. Wait for a policy engine route, polling until:
++
+[source,terminal]
+----
+$ POLICY_ENGINE_GRAPH_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.policyEngineURI}/api/upgrades_info/v1/graph}{"\n"}' updateservice example-name)"
+----
++
+succeeds.
+
+.. Retrieve a graph from the policy engine, polling until:
```

Comment:
I would like something that points out that the graph may not be available immediately after the route is available.  It can take some time for the update service pod to read in all the input data sources and construct its first graph.  But maybe we can idle this thread until we sort out [the earlier route-URI polling][1].

[1]: https://github.com/openshift/openshift-docs/pull/26219#discussion_r508004640

Reply:
I've pushed e747446 -> 6d253db, pivoting from asking the user to poll to giving them a POSIX-shell command that will handle the polling for them.